tools/accuracy_checker/accuracy_checker/metrics/im2latex_images_match.py

```diff
@@ -171,8 +177,13 @@ def render_routine(line):
         if not os.path.exists(pdf_filename):
             logging.info('ERROR: %s cannot compile\n', file_idx)
         else:
-            subprocess.run(['convert', '+profile', '"icc"', '-density', '200', '-quality', '100',
-                            pdf_filename, png_filename], check=True, stdout=PIPE, stderr=PIPE)
+            if os.name == 'nt':
+                subprocess.run(['convert', '+profile', '"icc"', '-density', '200', '-quality', '100',
+                                                  pdf_filename, png_filename],
+                                                 check=True, stdout=PIPE, stderr=PIPE, shell=True)
+            else:
+                subprocess.run(['convert', '+profile', '"icc"', '-density', '200', '-quality', '100',
+                                                  pdf_filename, png_filename], check=True, stdout=PIPE, stderr=PIPE)
```

Comment:
```suggestion
            subprocess.run(['convert', '+profile', '"icc"', '-density', '200', '-quality', '100',
                                                  pdf_filename, png_filename],
                                                 check=True, stdout=PIPE, stderr=PIPE, shell=True)
```
shell=True works in the same way on win and linux, so it is not problem to run with it always like in other subprocess calls above

Reply:
Merged commands